File: java/doit_java/old/chapter6/src/test/q5/CoffeeTest.java
package test.q5;

public class CoffeeTest {

    public static void main(String[] args) {

        Person personLim = new Person("임채영", 30000);
        Person personJung = new Person("정흥규", 50000);

        Starbucks starCoffee = new Starbucks();
        Coffeebean beanCoffee = new Coffeebean();

        personLim.buyStarCoffee(starCoffee, 4000);
        personJung.buyBeanCoffee(beanCoffee, 4500);
    }
}

File: python/python_oop_practice/p_oop_2/chap05/excercise/coffee/coffee_test.py
from person import Person
from star_coffee import StarCoffee
from bean_coffee import BeanCoffee
from menu import Menu


def main():
    kim = Person("Kim", 10000)
    lee = Person("Lee", 10000)

    star_coffee = StarCoffee()
    bean_coffee = BeanCoffee()

    kim.buy_star_coffee(star_coffee, Menu.STARAMERICANO)
    lee.buy_bean_coffee(bean_coffee, Menu.BEANLATTE)


if __name__ == '__main__':
    main()
